### PR TITLE
Feature/statistical measures

### PR DESCRIPTION
**Pull request by ckarczewski**

### CODE REVIEW

File: backend/zpiproject/urls.py [hunk 1]
```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', include('nbp.urls')),
```
Comment (ShanghaiG) [inline backend/zpiproject/urls.py:21]: why `admin/` path?

**Review by ShanghaiG — APPROVED** [review]

File: backend/zpiproject/urls.py (hunk 1 above)
Comment (ckarczewski) [reply backend/zpiproject/urls.py:21]: added by default, we do not need to use it

Comment (jowczar) [inline backend/zpiproject/urls.py:21]: I can't see it running on the server, perhaps it's not set-up correctly on Vercel?
@ShanghaiG can you check it, we have now index GET / route here to test the build on

File: backend/nbp/helpers/statistical_measures.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import math
+# test = [181, 187, 196, 196, 198, 203, 207, 211, 215, 123, 199]
```
Comment (jowczar) [inline backend/nbp/helpers/statistical_measures.py:2]: rm unused code

File: backend/tests/test_statistical_measures.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+from django.test import TestCase
```
Comment (jowczar) [inline backend/tests/test_statistical_measures.py:1]: I added unit tests & boundary value tests.
Overall:
– handle zeros and empty arrays (you can get them from db if e.g. currency is no longer used)
– set some specific .toFixed() decimal points amount so it's testable
– there can be many modes if numbers in dataset have same counts – every calculator returns it as an array []

These tests FAIL for now so @ShanghaiG you may want to disable them if you are doing sth on CI

**Review by jowczar — CHANGES_REQUESTED** [review]
Fix cases covered by tests

File: backend/nbp/views.py [hunk 4]
```diff
@@ -1,3 +1,34 @@
-from django.shortcuts import render
-
+from django.shortcuts import (HttpResponse)
+from django.http import JsonResponse
+from nbp.helpers.statistical_measures import median, dominant, standard_deviation, coefficient_of_variation
+from datetime import datetime, timedelta
+from nbp.models import TabelaA
 # Create your views here.
+
+def measures(request):
```
Comment (jowczar) [inline backend/nbp/views.py:8]: I will add subtask to Functional tests in Jira for testing this piece of code

**Review by jowczar — APPROVED** [review]
